### PR TITLE
Update mcr.cpp

### PR DESCRIPTION
**Pull request by zjb2016**

### CODE REVIEW

**Comment by KaiserHUA** [comment]
111

File: mcr.cpp [hunk 1]
```diff
@@ -1,125 +1,93 @@
-#include <iostream>
```
Comment (123sherlocklee) [inline mcr.cpp:1]: #test#review

**Review by zjb2016 — COMMENTED** [review]
hello I think because I reformatted the file , it seems change a lot

**Comment by Yuuksta** [comment]
lalalal

**Pull request merged**